esbuild.js

```diff
@@ -86,25 +93,24 @@ async function main() {
 
   warn('\n', { entryPoints })
 
-  try {
-    // eslint-disable-next-line  no-use-before-define
-    await builder()
-  } catch {
-    // It's common enough that `import https://esm.archive.org/lit` can _sometimes_ fail
-    // (seems like a race condition) maybe 10-25% of the time.  Retry once in case.
-    warn('\nsleeping 15s and retrying')
-    await exe('sleep 15')
+  for (let n = 0; n < MAX_RETRIES; n++) {
     try {
       // eslint-disable-next-line  no-use-before-define
       await builder()
-    } catch {
-      Deno.exit(1)
-    }
+
+      // build success
+      warn('\n[esbuild] done')
+      Deno.exit(0)
+      // eslint-disable-next-line no-empty
+    } catch {}
```

Comment:
the `builder()` method code is covered with a `.then().catch()` which seems to catch every issue so far -- and we `warn()` there w/ the caught exception.

so to avoid printing out the same exception (msg + stack trace) twice, i was omitting in that loop (and we'll always see the "sleeping 15s and retrying" message -- so even if something *somehow* threw an exception that wasnt printed in `builder()` -- we'd see the clear "uho, gonna sleep & loop" message